### PR TITLE
Replace sequence with REST implementation

### PR DESCRIPTION
**Pull request by houshengbo**

### CODE REVIEW

**Comment by houshengbo** [comment]
PG4 969 passed.

File: tests/src/test/scala/system/basic/WskConsoleTests.scala [hunk 1]
```diff
@@ -70,7 +70,18 @@ class WskConsoleTests extends TestHelpers with WskTestHelpers {
     val start = Instant.now.minusSeconds(5)
     val payload = new String("from the console!".getBytes, "UTF-8")
     val run = wsk.action.invoke(fullActionName, Map("payload" -> payload.toJson))
-    withActivation(wsk.activation, run, totalWait = 30 seconds) { activation =>
+
+    val duration = Duration(Instant.now.minusMillis(start.toEpochMilli).toEpochMilli, MILLISECONDS)
```
Comment (dubeejw) [inline tests/src/test/scala/system/basic/WskConsoleTests.scala]: Looks like this `duration` isn't used. the one below it is used.

File: tests/src/test/scala/common/rest/WskRest.scala [hunk 2]
```diff
@@ -1149,10 +1146,15 @@ class RunWskRestCmd() extends FlatSpec with RunWskCmd with Matchers with ScalaFu
   }
   val connectionContext = new HttpsConnectionContext(SSL.nonValidatingContext, Some(sslConfig))
 
-  def validateStatusCode(expectedExitCode: Int, statusCode: Int) = {
+  def validateStatusCode(expectedExitCode: Int, statusCode: Int, backupExpectedExitCode: Int = -1) = {
```
Comment (dubeejw) [inline tests/src/test/scala/common/rest/WskRest.scala]: What is the need for `backupExpectedExitCode`?

File: tests/src/test/scala/system/basic/WskConsoleTests.scala (hunk 1 above)
Comment (houshengbo) [reply tests/src/test/scala/system/basic/WskConsoleTests.scala]: Done.

File: tests/src/test/scala/common/rest/WskRest.scala (hunk 2 above)
Comment (houshengbo) [reply tests/src/test/scala/common/rest/WskRest.scala]: There is a special case for invokeaction.
If blocking or result is set to true, 200 is an acceptable status code, no matter what expectedExitCode is provided. I will change the code of invokeaction to narrate this case.

**Pull request merged**